shard-operator-parent/shard-operator-v2/src/test/java/com/redhat/service/smartevents/shard/operator/v2/ManagerClientTest.java

```diff
@@ -92,4 +89,26 @@ public void TestToNotifyBridgeStatus() {
                 .withRequestBody(equalToJson(expectedJsonUpdate, true, true))
                 .withHeader("Content-Type", equalTo("application/json")));
     }
+
+    @Test
+    public void TestToNotifyProcessorStatus() {
```

Comment:
If I was being picky I'd say the method name should be camel-case not pascal-case.

Reply:
done